sys/netbsd/sys_amd64.const

```diff
@@ -145,6 +156,7 @@ SYS_unlink = 10
 SYS_unlinkat = 471
 SYS_utimensat = 467
 SYS_utimes = 420
+SYS_vfork = 66
```

Comment:
There are two vfork calls, this is old one.

The newer one is:

```
/* syscall: "__vfork14" ret: "int" args: */
#define SYS___vfork14   282
```

Testing old is fine, but we need to check the new as well.